examples/react-component-reference/get-snapshot-before-update.js

```diff
@@ -3,19 +3,21 @@ class ScrollingList extends React.Component {
 
```

Comment:
Just realized we're using class property syntax above. These are stable docs so we shouldn't be.

Reply:
Thanks for changing this!